pkgs/by-name/k9/k9s/package.nix

```diff
@@ -19,14 +19,12 @@ buildGoModule rec {
     "-X github.com/derailed/k9s/cmd.date=1970-01-01T00:00:00Z"
   ];
 
-  tags = [ "netcgo" ];
```

Comment:
Sorry for keeping the draft state for long days. I understand that removing both netgo and netcgo might work on current golang.

However, I am unable to reproduce this problem on my devices. It requires k8s clusters via VPN and using it on macOS. Therefore, I will close this PR. 🙇‍♂️